src/main/java/bowling/domain/frame/Round.java

```diff
@@ -0,0 +1,92 @@
+package bowling.domain.frame;
+
+import bowling.domain.state.PitchState;
+import bowling.domain.state.Start;
+import bowling.domain.view.ResultView;
+
+public class Round {
+    public static int FINAL_MAX_COUNT = 3;
+
+    private int round;
+    private PitchState pitchState;
+
+    public Round(int round, PitchState pitchState) {
+        this.round = round;
+        this.pitchState = pitchState;
+    }
+
+    public static Round createOf() {
+        return new Round(ResultView.MIN_FRAME, new Start());
+    }
+
+    public static Round createOf(int round) {
+        return new Round(round, new Start());
+    }
+
+    public Round next(int point) {
+        pitchState = pitchState.next(point);
+        return new Round(round, pitchState);
+    }
+
+    public Round nextStart() {
+        return new Round(round + 1, new Start());
+    }
+
+    public Round next() {
+        return new Round(round + 1, pitchState);
+    }
+
+    public boolean isNormalMaxRound() {
+        return round == ResultView.MAX_FRAME - 1;
+    }
+
+    public boolean isFinalMaxRound() {
+        if (!isNext() && isFinal()) {
+            return true;
+        }
+
+        return round > FINAL_MAX_COUNT ;
+    }
+
+    public boolean isNext() {
```

Comment:
pitch를 해야하는지 반환하는 메서드인데요. hasNext()라는 네이밍은 어떤가요?
```suggestion
    public boolean hasNext() {
```